src/ng/sce.js

```diff
@@ -155,97 +155,101 @@ function adjustMatchers(matchers) {
+   * **Note:** the default `trustedResourceUrlList` of 'self' is not recommended if your app shares
+   * its origin with other apps! It is a good idea to limit it to only your application's directory.
    * </div>
    */
-  this.resourceUrlWhitelist = function(value) {
+  this.trustedResourceUrlList = function(value) {
     if (arguments.length) {
-      resourceUrlWhitelist = adjustMatchers(value);
+      trustedResourceUrlList = adjustMatchers(value);
     }
-    return resourceUrlWhitelist;
+    return trustedResourceUrlList;
   };
+  this.resourceUrlWhitelist = this.trustedResourceUrlList;
 
   /**
    * @ngdoc method
-   * @name $sceDelegateProvider#resourceUrlBlacklist
+   * @name $sceDelegateProvider#bannedResourceUrlList
    * @kind function
    *
-   * @param {Array=} blacklist When provided, replaces the resourceUrlBlacklist with the value
-   *     provided.  This must be an array or null.  A snapshot of this array is used so further
-   *     changes to the array are ignored.</p><p>
+   * @param {Array=} bannedResourceUrlList When provided, replaces the `bannedResourceUrlList` with
+   *     the value provided. This must be an array or null. A snapshot of this array is used so
+   *     further changes to the array are ignored.</p><p>
    *     Follow {@link ng.$sce#resourceUrlPatternItem this link} for a description of the items
    *     allowed in this array.</p><p>
-   *     The typical usage for the blacklist is to **block
+   *     The typical usage for the `bannedResourceUrlList` is to **block
    *     [open redirects](http://cwe.mitre.org/data/definitions/601.html)** served by your domain as
    *     these would otherwise be trusted but actually return content from the redirected domain.
    *     </p><p>
-   *     Finally, **the blacklist overrides the whitelist** and has the final say.
+   *     Finally, **the banned resource URL list overrides the trusted resource URL list** and has
+   *     the final say.
    *
-   * @return {Array} The currently set blacklist array.
+   * @return {Array} The currently set `bannedResourceUrlList` array.
    *
    * @description
-   * Sets/Gets the blacklist of trusted resource URLs.
+   * Sets/Gets the `bannedResourceUrlList` of trusted resource URLs.
    *
-   * The **default value** when no whitelist has been explicitly set is the empty array (i.e. there
-   * is no blacklist.)
+   * The **default value** when no trusted resource URL list has been explicitly set is the empty
+   * array (i.e. there is no `bannedResourceUrlList`.)
    */
 
-  this.resourceUrlBlacklist = function(value) {
+  this.bannedResourceUrlList = function(value) {
     if (arguments.length) {
-      resourceUrlBlacklist = adjustMatchers(value);
+      bannedResourceUrlList = adjustMatchers(value);
     }
-    return resourceUrlBlacklist;
+    return bannedResourceUrlList;
   };
+  this.resourceUrlBlacklist = this.bannedResourceUrlList;
```

Comment:
No getter/setter for this one?

Reply:
I don't know why I thought this was a private api, fixed.